front/src/components/app/index.js

```diff
@@ -13,6 +14,7 @@ class App extends Component {
                     <Route path="/student-registration/:token" component={StudentRegistrationPage} />
                     <Route path="/teachers" exact component={TeachersPage} />
                     <Route path="/login" component={LoginPage} />
+                    <Route path="/dashboard" component={UpdateProfile} />
```

Comment:
It should be a modal as a part of the header and be available on any page. It should not be a separated route

Reply:
We don't have header in our app yet, that's why I render this modal on a separated route